framework/widgets/Menu.php

```diff
@@ -210,9 +210,12 @@ protected function renderItems($items)
             }
             if (!empty($class)) {
                 if (empty($options['class'])) {
-                    $options['class'] = implode(' ', $class);
+                    $options['class'] = $class;
                 } else {
-                    $options['class'] .= ' ' . implode(' ', $class);
+                    if (!is_array($options['class'])) {
```

Comment:
I feel like the latter is an over complication of the merge statement. I used the former approach and added a comment clarifying why this is needed. What do you think?